cms/models.py

```diff
@@ -684,11 +690,18 @@ def products(self):
         return page_data
 
     def get_context(self, request, *args, **kwargs):
+        posthog = Posthog(settings.POSTHOG_API_TOKEN, host=settings.POSTHOG_API_HOST)
+        show_new_featured_carousel = posthog.feature_enabled(
+            "mitxonline-new-featured-carousel",
+            "randomID",
+            person_properties={"environment": settings.ENVIRONMENT},
+        )
```

Comment:
The only time we really care about the flag is on the front end - but the django template and feature flags don't get along all the way, so I'm passing from front to back end rather than loading on the html page with vanilla js or changing post-page load